devdoc/WorkUnits.rst

```diff
@@ -430,12 +430,47 @@ Once all of the required workflow items are executed, the workunit is marked as
 workflow items that are waiting to be triggered by an event, the workunit will be passed to the scheduler,
 which will keep monitoring for events.
 
+Note that most items in the xml workflow begin in the state WFStateNull. This means that it is valid to execute them, but they haven't been executed yet. Typically, only a few items begin in the state required.
```

Comment:
minor: most of the text in this file tries to limit the line length to ~120 characters to make it easier to read if line wrapping is not enabled.  Please can you split up the few long lines.